### PR TITLE
Add rating function

### PR DESCRIPTION
**Pull request by m1ga**
#11 

* Added function to calculate the rating; return as String
* Added example to the README

### CODE REVIEW

**Comment by Fubinator** [comment]
Hey, it seems like the action is not working properly, but can you run `yarn lint` locally and fix those issues?

File: lib/cvss.js [hunk 1]
```diff
@@ -36,6 +36,27 @@ function CVSS(vector) {
...
+  /**
+   * Parses the vector to a rating
+   *
+   * @returns {String} Rating value
+   */
+  function getRating() {
+    const score = getScore();
+    let rating = "None";
+
+    if (score <= 3.9) {
```
Comment (Fubinator) [inline lib/cvss.js:48]: I think you need to check if the score is `> 0` here, because with a score of `<= 0` it returns `Low` instead of `None`

Comment (m1ga) [reply lib/cvss.js]: sorry about that! Tested all but 0 :smile: Fixed now!
0 -> None
0.1 -> Low

and fixed the linting parts!

**Review by Fubinator — APPROVED** [review]

**Comment by Fubinator** [comment]
LGTM thanks for you contribution!

**Pull request merged**